### ISSUE TITLE
train pose estimation from strach

### ISSUE THREAD

**Issue by Dantju** [xingyizhou/CenterNet#589]
@xingyizhou ,have u train PE from strach but not finetune on OD? I did this ,and get a better result，i want to know,why u train PE finetuned on OD?

**Comment by Dantju** [comment]
train pose estimation from scratch, sorry get a spelling error

**Comment by xingyizhou** [comment]
Thanks for raising this up. I have tried and also observed better results. The new results will be updated in the next revision.

**Comment by Dantju** [comment]
@xingyizhou I observed a better result on PE ,but PD get a lit worse,do u have some idear to improve PD?

**Comment by Dantju** [comment]
@xingyizhou I find use multi_pose to train PD by setting PE's loss weight as 0，the result is worse than using ctdet to train PD ,do u have any idea?

**Comment by sisrfeng** [comment]
Hi, I train dla_34 and resdcn_18 on UA-DETRAC using ImageNet pretrained backbone but not loading coco-pretrained model.
The AP50 is always 0 for 50+ epoch.
Do you know how to solve it?
Thx!

> @xingyizhou I find use multi_pose to train PD by setting PE's loss weight as 0，the result is worse than using ctdet to train PD ,do u have any idea?
